### PR TITLE
Moving layout and colors toward new approved designs. Moving checkout…

### PR DESCRIPTION
**Pull request by mcstover**
… back into a modal. Setting up temporary show checkout button with minimized status component.

### CODE REVIEW

File: src/components/CorporateCampaign/CampaignHeader.vue [hunk 1]
```diff
@@ -1,36 +1,58 @@
...
+	<section
+		class="campaign-header section"
+	>
+		<div class="row align-center">
+			<div class="small-12 medium-10 large-6 xlarge-5 small-order-2 large-order-1 columns">
+				<!-- <featured-loans-carousel
+					:disable-redirects="true"
+					:show-view-loan-cta="false"
+					@add-to-basket="handleAddToBasket"
+				/> -->
```
Comment (mcstover) [inline src/components/CorporateCampaign/CampaignHeader.vue:11]: we'll bring this in once we create a LoanCardController version of the NoClickLoanCard.

File: src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue [hunk 2]
```diff
@@ -75,15 +126,19 @@ import numeral from 'numeral';
...
 import checkoutUtils from '@/plugins/checkout-utils-mixin';
-import { blueHeader, blueFooter } from '@/util/siteThemes';
+import { lightHeader, lightFooter } from '@/util/siteThemes';
 import cookieStore from '@/util/cookieStore';
 import CampaignHeader from '@/components/CorporateCampaign/CampaignHeader';
 import CampaignHowKivaWorks from '@/components/CorporateCampaign/CampaignHowKivaWorks';
-import CampaignLoanDisplay from '@/components/CorporateCampaign/CampaignLoanDisplay';
+import CampaignLoanGridDisplay from '@/components/CorporateCampaign/CampaignLoanGridDisplay';
+import CampaignLoanRow from '@/components/CorporateCampaign/CampaignLoanRow';
+// import CampaignLoanRowsDisplay from '@/components/CorporateCampaign/CampaignLoanRowsDisplay';
```
Comment (mcstover) [inline src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue:135]: Future container for multiple instances of a Loan row...

File: src/components/CorporateCampaign/CampaignLoanRow.vue [hunk 3]
```diff
@@ -0,0 +1,406 @@
+<template>
```
Comment (mcstover) [inline src/components/CorporateCampaign/CampaignLoanRow.vue:1]: This is a highly modified version of the category row used on the homepage which is not powered by a loan channel and mean to prototype the layout. Eventual goal is to allow 1 type of filter per row with a friendly header and selector for the target facet. ie. "Find loans in ##SomeCountry##". Additionally, it has a "load more" button to just keep adding loans to the row.

**Review by eddieferrer — APPROVED** [review]

File: src/components/CorporateCampaign/CampaignLoanRow.vue (hunk 3 above)
Comment (ryan-ludwig) [inline src/components/CorporateCampaign/CampaignLoanRow.vue:1]: We should probably convert this into a `kv-carousel`. The original homepage loan carousel was built before `kv-carousel`, and hasn't been swapped over yet.

**Review by ryan-ludwig — APPROVED** [review]

File: src/components/CorporateCampaign/CampaignLoanRow.vue (hunk 3 above)
Comment (mcstover) [reply src/components/CorporateCampaign/CampaignLoanRow.vue:1]: I did wonder about that and it's an improvement we can make if there is time, but it was already working an I wanted to start this prototyping work...

**Pull request merged**